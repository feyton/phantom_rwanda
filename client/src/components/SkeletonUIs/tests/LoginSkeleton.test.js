import React from 'react';
import renderer from 'react-test-renderer';
import LoginSkeleton from '../LoginSkeleton.js';

describe('LanguageButton', () => {
  it('should render LanguageButton', () => {
    const loginSkeleton = renderer.create(<LoginSkeleton />).toJSON();
    expect(loginSkeleton).toMatchSnapshot();
  });
});
