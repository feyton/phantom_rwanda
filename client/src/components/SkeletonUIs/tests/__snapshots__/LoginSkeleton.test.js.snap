// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageButton should render LanguageButton 1`] = `
<div
  className="flex flex-col animate-pulse"
>
  <main
    className="flex flex-col  md:flex-row items-center  justify-center relative h-full"
  >
    <section
      className="hidden md:flex flex-col  py-3 px-4 w-72 md:w-fit md:min-w-[300px] absolute  md:top-0 md:mx-auto md:relative rounded-lg"
    >
      <section
        className="flex flex-col  w-full"
      >
        <div
          className="bg-gray-100 w-44 h-6 mb-5"
        />
        <div
          className="bg-gray-100 w-60 h-8 mb-7"
        />
        <div
          className="bg-gray-100 w-60 h-8 mb-7"
        />
        <div
          className="bg-gray-100 w-20 h-8 rounded-md mb-7"
        />
      </section>
    </section>
  </main>
</div>
`;
